### PR TITLE
fix: support shufflenet

### LINKED ISSUE #486: ↔ [Converter] Add support for aten::chunk in TRTorch
##

- **Function Schema**: `aten::chunk(Tensor(a) self, int chunks, int dim=0) -> (Tensor[])`

- **Original PyTorch API**: `torch.chunk(input, chunks, dim=0) → List of Tensors`

### PR DESCRIPTION
**Pull request by letian-jiang**
# Description

This PR includes following changes 
* `chunk.cpp`: add a converter of `aten::chunk` op
* `conv_deconv.cpp`: use `ITensorOrFreeze(ctx)` to pull ITensor
* `prim.cpp`: fix errors in cases where Var is IValue containing a ITensor

After above changes, the shufflenet can be compiled successfully 
```
import torch
import torchvision
import trtorch

model = torchvision.models.shufflenet_v2_x1_0(pretrained=True).cuda().eval()
input_data = torch.randn(1, 3, 224, 224).cuda()
scripted_model = torch.jit.script(model)
out = scripted_model(input_data)

compile_settings = {
    "input_shapes": [(1, 3, 224, 224)],
}

trt_ts_module = trtorch.compile(scripted_model, compile_settings)
```

Fixes #486 

## Type of change

- Bug fix (non-breaking change which fixes an issue)

# Checklist:

- [x] My code follows the style guidelines of this project (You can use the linters)
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas and hacks
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests to verify my fix or my feature
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by narendasan** [comment]
Hi, most of this looks good. We will review the changes. The first thing we need to do though is get your commits signed properly in accordance with the DCO (https://www.github.com/NVIDIA/TRTorch/blob/master/CONTRIBUTING.md)

File: core/conversion/evaluators/prim.cpp [hunk 1]
```diff
@@ -95,6 +95,11 @@ auto prim_registrations =
                             tensor_holder.hold_tensor(args.at(in).ITensor());
                             auto ival = c10::IValue(std::move(c10::make_intrusive<TensorContainer>(tensor_holder)));
                             list.emplace_back(std::move(ival));
+                          } else if (args.at(in).isIValue() && args.at(in).IValue()->isCustomClass()){
+                            auto tensor_holder = TensorContainer();
+                            tensor_holder.hold_tensor(args.at(in).IValue()->toCustomClass<TensorContainer>().get()->tensor());
+                            auto ival = c10::IValue(std::move(c10::make_intrusive<TensorContainer>(tensor_holder)));
+                            list.emplace_back(std::move(ival));
```
Comment (peri044) [inline core/conversion/evaluators/prim.cpp:102]: Can you add a test case for this covering aten::chunk ? Please refer to our converter test suite https://github.com/NVIDIA/TRTorch/tree/master/tests/core/conversion/converters

**Review by peri044 — CHANGES_REQUESTED** [review]
